src/extension.ts

```diff
@@ -92,6 +92,10 @@ export async function activate(context: vscode.ExtensionContext) {
         handleError(error, info.id, defaultMessage)
     })
 
+    if (isCloud9()) {
+        vscode.window.withProgress = wrapWithProgressForCloud9(globals.outputChannel)
+    }
+
```

Comment:
This is the only non-test related change. Previously the output channel was wrapped for VSC and C9. Now it's just C9.